samples/createNote.js

```diff
@@ -14,8 +14,8 @@ async function main(
   // const noteId = 'my-note-id' // Id of the note
 
   // Import the library and create a client
-  const containerAnalysis = require('@google-cloud/containeranalysis');
-  const client = new containerAnalysis.v1beta1.GrafeasV1Beta1Client();
+  const { GrafeasClient } = require('@google-cloud/containeranalysis');
+  const client = new GrafeasClient();
```

Comment:
So are we not using the container analysis client at all then? Wouldn't authentication be complex using the grafeas client?